django/urls/resolvers.py

```diff
@@ -609,9 +609,10 @@ def url_patterns(self):
             iter(patterns)
         except TypeError as e:
             msg = (
-                "The included URLconf '{name}' does not appear to have any "
-                "patterns in it. If you see valid patterns in the file then "
-                "the issue is probably caused by a circular import."
+                "The included URLconf '{name}' does not appear to have "
+                "any patterns in it or maybe you mispelled the "
+                "'urlpatterns' variable. If you see valid patterns in the "
+                "file then the issue is probably caused by a circular import."
```

Comment:
I think it's too detailed, maybe:
```suggestion
                "The included URLconf '{name}' does not appear to have "
                "any patterns in it. If you see the 'urlpatterns' variable "
                "with valid patterns in the file then the issue is probably "
                "caused by a circular import."
```

Reply:
Sorry, I commited this conversation unintentionally and seems that the tests did not pass. Should I commit this?